interfaces/cython/cantera/test/test_onedim.py

```diff
@@ -449,6 +449,14 @@ def test_refine_criteria_boundscheck(self):
                 vals[i] = bad[i]
                 self.sim.set_refine_criteria(*vals)
 
+    def test_refine_criteria(self):
+        self.create_sim(ct.one_atm, 300.0, 'H2:1.1, O2:1, AR:5')
+        vals = [3.0, 0.1, 0.2, 0.05]
+        self.sim.set_refine_criteria(*vals)
+        check = self.sim.get_refine_criteria()
+        for v, c in zip(vals, check):
+            assert v == c
```

Comment:
Should be `self.assertEqual(v, c)` (or even better, ditch the loop and write `self.assertEqual(vals, check)`)